Completed signup page

Sandaru-Dev opened the pull request:
For this pull request, I made the SignUp page completely.

Fixes issue SignUp page #38

![SignUp Page](https://github.com/MadhushaPrasad/Diagon-Alley/assets/77701910/bdbfe593-f910-43ab-9d62-00f307f23086)

Sandaru-Dev commented:
Hi @MadhushaPrasad , This PR is ready for review. Thank you!

supuna97 commented on pages/signup.php:
```diff
@@ -0,0 +1,85 @@
+                <label for="">Confirm Password</label>
+                <input type="password" name="" id="" placeholder="********">
+            </div>
+
+        
```
Remove spaces. Follow coding best practices.

Sandaru-Dev replied:
Thank you so much for the comment. I will remove the spaces and will follow the coating practices.

MadhushaPrasad commented on pages/signup.php:
```diff
@@ -0,0 +1,74 @@
+                <label for="Student">Admin</label>
+            </div>
+            
+            <div class="termscondition">
```
Can you please change this CSS class name to camelCase or hyphens ?

Sandaru-Dev replied:
Thank you so much for your comments, Madhusha. I will revise the code and push it shortly.

MadhushaPrasad commented on pages/signup.php:
```diff
@@ -0,0 +1,74 @@
+                <hr>
+            </div>
+
+            <div class="socialmedia">
```
this one also needs to be converted to camelCase or hyphens.

MadhushaPrasad reviewed (changes requested):
Mobile responsive is not working properly.

![image](https://github.com/MadhushaPrasad/Diagon-Alley/assets/50085447/27fe5a64-7c90-423c-931f-aa703814b391) - (width:425)

![image](https://github.com/MadhushaPrasad/Diagon-Alley/assets/50085447/b1d1646e-3b89-48c3-aa46-aa0db756f38f) - tablet view (width:768)

So please check responsive for every devices.

supuna97 commented on pages/signup.css:
```diff
@@ -0,0 +1,289 @@
+
+.position-fixed{
+    position: fixed;
+    /* text-align: center; */
```
Remove commented code lines.

supuna97 commented on pages/signup.css:
```diff
@@ -0,0 +1,289 @@
+    }
+
+    .text-content h4{
+        /* width: 50px; */
```
Remove unwanted code lines

supuna97 commented on pages/signup.css:
```diff
@@ -0,0 +1,289 @@
+}
+
+@media screen and (max-width: 468px) {
+
```
Remove unwant empty lines.

supuna97 commented on pages/signup.css:
```diff
@@ -0,0 +1,289 @@
+}
+
+@media screen and (max-width: 900px) {
+
```
Remove unwant empty lines

supuna97 commented on pages/signup.css:
```diff
@@ -0,0 +1,289 @@
+    .component-2 {
+        display: none;
+    }
+
```
Remove unwant empty lines

supuna97 commented on pages/signup.css:
```diff
@@ -0,0 +1,289 @@
+}
+
+@media screen and (max-width: 1068px) {
+
```
Remove unwant empty lines

supuna97 commented on pages/signup.css:
```diff
@@ -0,0 +1,289 @@
+
+}
+.social-media a{
+    /* padding: 8px 30px; */
```
Remove commented lines

Sandaru-Dev replied:
Thank you, @supuna97, for your valuable comments. I will make the edits you mentioned. 😊

supuna97 commented:
Hi @Sandaru-Dev  Please check css files and remove unnecessary empty lines. No need for empty lines between css classes.

MadhushaPrasad commented on pages/signup.php:
```diff
@@ -0,0 +1,74 @@
+
+            <div class="form">
+                <label for="">Email</label>
+                <input type="email" name="" id="" placeholder="mail@example.com">
```
if you don't use an id for an element then remove this. empty value attributes are not good for security and as well as best practices.

IDs must be unique, but the validator sees id="" as having the same value. So this is bad. please remove those.

MadhushaPrasad commented on pages/signup.php:
```diff
@@ -0,0 +1,74 @@
+                <input type="email" name="" id="" placeholder="mail@example.com">
+
+                <label for="">User Name</label>
+                <input type="text" name="" id="" placeholder="madhush99">
```
id issue

MadhushaPrasad commented on pages/signup.php:
```diff
@@ -0,0 +1,74 @@
+                <input type="text" name="" id="" placeholder="madhush99">
+
+                <label for="">Password</label>
+                <input type="password" name="" id="" placeholder="********">
```
- id issue

- if you use it `for` an attribute then you have to provide that, "for what?" it means you have to tell this for a specific element unique ID. So if you don't want a for attribute then please remove it, if not provide an ID for it.

MadhushaPrasad commented on pages/signup.php:
```diff
@@ -0,0 +1,74 @@
+                <input type="password" name="" id="" placeholder="********">
+
+                <label for="">Confirm Password</label>
+                <input type="password" name="" id="" placeholder="********">
```
- for issue
- id issue

MadhushaPrasad commented on pages/signup.php:
```diff
@@ -0,0 +1,74 @@
+            </div>
+            
+            <div class="terms-condition">
+                <input type="checkbox" name="" id="">
```
Please provide a value for every `name` attribute that is inside the form tag input elements.
- id issue

MadhushaPrasad commented on pages/signup.php:
```diff
@@ -0,0 +1,74 @@
+            </div>
+
+            <div class="social-media">
+                <a href="#"> <img src="../assets/img/google-icon.png" alt="">Sign up with Google</a>
```
Add the value for `alt` which represents the basic idea of the image

MadhushaPrasad commented on pages/signup.php:
```diff
@@ -0,0 +1,74 @@
+
+            <div class="social-media">
+                <a href="#"> <img src="../assets/img/google-icon.png" alt="">Sign up with Google</a>
+                <a href="#"><img src="../assets/img/facebook-icon.png" alt=""> Sign up with Facebook</a>
```
Add the value for `alt` which represents the basic idea of the image

Sandaru-Dev replied:
Hi @MadhushaPrasad, Thank you so much for your valuable comments. I will make the modification as you mentioned above. 
Thanks again!

MadhushaPrasad reviewed (changes requested):
Original signup page image :
![image](https://github.com/MadhushaPrasad/Diagon-Alley/assets/50085447/545aa8ca-dad1-44a8-98a2-6d03dacccb66)
- missed some elements and have an additional space bottom of the window
- can't see password elements eye icon both of password and confirm password.
- 
![image](https://github.com/MadhushaPrasad/Diagon-Alley/assets/50085447/ebfef0c4-4445-4e32-b95e-648308b40ce0)

tablet view : 
![image](https://github.com/MadhushaPrasad/Diagon-Alley/assets/50085447/1f6b2749-f4db-4fc3-9fb2-d5514e8be48e)
- all elemets need to be center.

laptop 1024 view :
![image](https://github.com/MadhushaPrasad/Diagon-Alley/assets/50085447/62619c0f-8918-4a86-ac25-7d5d4b6e5f3f)
- little bit deferent than the Figma view

Sandaru-Dev commented:
Hi @MadhushaPrasad, Thank you so much for your comments. I'm working on addressing those issues and will push the updates as soon as possible. Thank you!

MadhushaPrasad commented:
> Hi @MadhushaPrasad, Thank you so much for your comments. I'm working on addressing those issues and will push the updates as soon as possible. Thank you!

any update on this PR??

Sandaru-Dev commented:
Hi @MadhushaPrasad, Thank you so much for your comment. I'm working on the necessary changes and will push them as soon as possible. Thank you!

Sandaru-Dev commented:
Hi @MadhushaPrasad, My PR is waiting for your review. :)

MadhushaPrasad commented:
> Hi @MadhushaPrasad, My PR is waiting for your review. :)

sure,i will merge this🫡

MadhushaPrasad reviewed (approved):
great job🔥

Sandaru-Dev commented:
Thank you so much Madhusha. 😊
